dev/Code/CryEngine/RenderDll/Common/RenderThread.cpp

```diff
@@ -3463,6 +3466,8 @@ bool SRenderThread::IsFailed()
 bool CRenderer::FlushRTCommands(bool bWait, bool bImmediatelly, bool bForce)
 {
     AZ_TRACE_METHOD();
+    AZ_PROFILE_FUNCTION_STALL(AZ::Debug::ProfileCategory::Renderer);
```

Comment:
Added a profile marker so the main thread stalling on the render thread is visible in the profiler.